backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3333,4 +3333,102 @@ databaseChangeLog:
                   constraints:
                     nullable: true
                     foreignKeyName: fk__facility__default_specimen_type
-                    references: specimen_type(internal_id)
\ No newline at end of file
+                    references: specimen_type(internal_id)
+  - changeSet:
+      id: add-disease-table
+      author: emma@skylight.digital
+      comment: Adds a non-audited table to track which diseases SimpleReport supports
+      changes:
+        - tagDatabase:
+            tag: add-supported-disease-table
+        - createTable:
+            tableName: supported_diseases
```

Comment:
should this be plural or singular

Reply:
I think singular matches our patterns elsewhere in the db; changed to be singular across all changes